src/extension.ts

```diff
@@ -253,6 +259,18 @@ export class Extension implements RunHooks {
     this._context.subscriptions.push(this);
   }
 
+  private async _pickTraceFile() {
+    const vscode = this._vscode;
+    const selectedUris = await vscode.window.showOpenDialog({
+      canSelectMany: false,
+      filters: {
+        'Traces': ['zip'],
```

Comment:
Time for us to claim .pwtrace upstream, so that we could just add a VS Code viewer.

Reply:
I didn't understand what should I do there, can you clarify?

Reply:
That was not meant as a call for changes. I was saying that if we had a better extension, `foo.pwtrace`, we would be able to implement a VS Code viewer for it.